app/controllers/api/exchange_products_controller.rb

```diff
@@ -1,9 +1,10 @@
 # frozen_string_literal: true
 
 # This controller lists products that can be added to an exchange
+#
+# Pagination is optional and can be required by using param[:page]
 module Api
   class ExchangeProductsController < Api::BaseController
-    DEFAULT_PAGE = 1
```

Comment:
This is still used elsewhere in the class, right?

Reply:
not any more. If no page number is provided in the request, it will not paginate results. That's what the PR changes.

Reply:
My mistake, there's one occurrence. I need to remote it, it's not needed now.

Reply:
ok, fixed 👍